masterlist.yaml

```
@@ -5012,6 +5012,74 @@ plugins:
   - name: 'Project Nevada - Med-X Fix.esp'
     tag: [ Names ]
 
+# Mothership Zeta Rehaul
+  - name: '(Mothership)?ZetaRehaul((BiWastelands|DelayDLCRedux|HardcoreZeta|HardcoreZetaDelayDLCRedux|NVSpeechChecks)Patch)?\.es[mp]'
+    url:
+      - link: 'https://www.nexusmods.com/newvegas/mods/77340/'
+        name: 'TTW - Mothership Zeta Rehaul'
+  - name: 'MothershipZetaRehaul.esm'
+    msg:
+      - <<: *patchProvided
+        subs: [ 'Tale of Bi Wastelands TTW' ]
+        condition: 'active("Tale of Bi Wastelands.esm") and not active("ZetaRehaulBiWastelandsPatch.esp")'
+      - <<: *patchProvided
+        subs: [ 'Delay DLC Redux TTW' ]
+        condition: 'active("DelayDLCRedux.esp") and not active("HardcoreZeta.esp") and not active("ZetaRehaulDelayDLCReduxPatch.esp")'
+      - <<: *patchProvided
+        subs: [ 'TTW Hardcore Zeta' ]
+        condition: 'active("HardcoreZeta.esp") and not active("DelayDLCRedux.esp") and not active("ZetaRehaulHardcoreZetaPatch.esp")'
+      - <<: *patchProvided
+        subs: [ 'Delay DLC Redux Hardcore Zeta' ]
+        condition: 'active("HardcoreZeta.esp") and active("DelayDLCRedux.esp") and not active("ZetaRehaulHardcoreZetaDelayDLCReduxPatch.esp")'
+      - <<: *patchProvided
+        subs: [ 'TTW New Vegas Speech Checks' ]
+        condition: 'active("TTW New Vegas Speech Checks.esm") and not active("ZetaRehaulNVSpeechChecksPatch.esp")'
+    tag:
+      - Delev
+      - Enchantments
+      - Faction
+      - Invent.Remove
+      - Invent.Add
+      - Names
+      - Scripts
+    inc: [ 'TTWZetaRewards.esp' ]
+    dirty:
+      # Ver 3.5
+      - <<: *quickClean
+        crc: 0x782D858F
+        util: '[FNVEdit v4.0.4](https://www.nexusmods.com/newvegas/mods/34703)'
+        itm: 2
+  - name: 'ZetaRehaulBiWastelandsPatch.esp'
+    clean:
+      - crc: 0x1D82AD4C
+        util: 'FNVEdit v4.0.4'
+  - name: 'ZetaRehaulDelayDLCReduxPatch.esp'
+    msg:
+      - <<: *alreadyInX
+        subs: [ 'ZetaRehaulHardcoreZetaDelayDLCReduxPatch' ]
+        condition: 'active("ZetaRehaulHardcoreZetaDelayDLCReduxPatch.esp")'
```

The message could also be applied to `ZetaRehaulHardcoreZetaPatch.esp`.
Or is it still required?
In the subs, you can use `TTW Hardcore Zeta & Delay DLC Redux Patch` or the full filename `ZetaRehaulHardcoreZetaDelayDLCReduxPatch.esp`.
Either is fine, I'd go with whichever is easier to read.

```suggestion
  - name: 'ZetaRehaul(DelayDLCRedux|HardcoreZeta)Patch\.esp'
    msg:
      - <<: *alreadyInX
        subs: [ 'TTW Hardcore Zeta & Delay DLC Redux Patch' ]
        condition: 'active("ZetaRehaulHardcoreZetaDelayDLCReduxPatch.esp") and active("HardcoreZeta.esp") and active("DelayDLCRedux.esp")'
  - name: 'ZetaRehaulDelayDLCReduxPatch.esp'
```